HackYourFuture-CPH/fp-class12
# Add docker configuration

# Description

This enables you to optionally run your database in Docker.



Fixes # (issue)

# How to test?

Make sure your `.env` file is filled out correctly and run `docker-compose up -d` to fire up a container running Mysql.

# Checklist

- [x] I have merged the latest version of the source branch (typically develop) into this branch before pushing my changes
